protocol/writing-resource/post-target-not-found.feature

```diff
@@ -0,0 +1,43 @@
+Feature: POST to non-existing resource must result in 404
+
+  Background: Set up container
+    * def testContainer = rootTestContainer.createContainer()
+    * def container = testContainer.reserveContainer()  
+    * def rdfResource = testContainer.reserveResource('.ttl');
+    * def jsonResource = testContainer.reserveResource('.json');
+    * def fooResource = testContainer.reserveResource('.foo');
+    # TODO: Is it better to reserve these in the scenarios?
+
+  Scenario: Reserved container does not exist
+    Given url container.url
+    And headers clients.alice.getAuthHeaders('POST', container.url)
+    And header Content-Type = 'text/turtle'
+    When method POST
+    Then status 404
+    
+  Scenario: Reserved RDF resource does not exist
+    Given url rdfResource.url
+    And headers clients.alice.getAuthHeaders('POST', rdfResource.url)
+    And header Content-Type = 'text/turtle'
+    When method POST
+    Then status 404
+
+  Scenario: Reserved RDF resource still does not exist
```

Comment:
The title makes me think this test is part of the previous scenario but it will actually run in parallel since it is a separate one. It should still succeed as the resource should not exist but the word `still` is then misleading. To make it a follow on step, remove this line (25).

Reply:
Should this extra test be repeated for each scenario i.e. the server should return a 404 and we also check nothing was created.

Reply:
Yes, those are both good points, I will take that into account.